scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -632,23 +637,23 @@ task LoadTable {
     String datatype
     String superpartitioned
     String? schema
-    File? service_account_json
+    String? service_account_json_path
     String table_creation_done
     Array[String] tsv_creation_done
     String run_uuid
 
     String docker
   }
 
-  String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
 
   command <<<
     set -x
     set -e
 
     if [ ~{has_service_account_file} = 'true' ]; then
-      gcloud auth activate-service-account --key-file='~{service_account_json}'
-      gcloud config set project ~{project_id}
```

Comment:
why is the project id not being set anymore?

Reply:
whoops — good catch!  should be back in